<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vertical" stretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Laksaman&quot;;
</string>
      </property>
      <property name="text">
       <string>Maydala Panel</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="horizontalWidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	height:4em;
	margin:5px;
	font: 14pt &quot;Laksaman&quot;;
	border:1px solid ;
	border-radius:8px;
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.00526316 rgba(53, 240, 61, 255), stop:1 rgba(46, 127, 233, 255));
}

QPushButton:hover {
	border-radius:15px;
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="person_btn">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../images/customers.png</normaloff>../images/customers.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="orders_btn">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../images/carts.png</normaloff>../images/carts.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="products_btn">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../images/products.png</normaloff>../images/products.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
